jbmc/src/java_bytecode/library/CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
+message(STATUS "Downloading java-models-library...")
+include("${CBMC_SOURCE_DIR}/../cmake/DownloadProject.cmake")
+download_project(PROJ java_models_library
+    URL https://github.com/diffblue/java-models-library/archive/master.zip
+    PATCH_COMMAND cmake -E copy_directory "${JBMC_BINARY_DIR}/../../java_models_library-src/src"
```

Comment:
Please document why you use the `PATCH_COMMAND` ;)

Reply:
done